pkg/pillar/worker/workerpool.go

```diff
@@ -172,6 +194,12 @@ func (wp *Pool) MsgChan() <-chan Processor {
 	return wp.resultChan
 }
 
+// C returns a channel to be used in a select loop.
+// This is a duplicate of MsgChan
```

Comment:
Can we remove the C() function and only have MsgChan() for Single and Pool?

Reply:
Way back when, I think this was just `MsgChan`, and we kept/added `C` to line it up with what other things did; it just was cleaner. I don't think anything uses it, but it is common.

Reply:
OK